sway/ipc-json.c

```diff
@@ -331,6 +333,66 @@ static void ipc_json_describe_enabled_output(struct sway_output *output,
 	json_object_object_add(object, "adaptive_sync_status",
 		json_object_new_string(adaptive_sync_status));
 
+	struct json_object *layers = json_object_new_array();
+	size_t len = sizeof(output->layers) / sizeof(output->layers[0]);
+	for (size_t i = 0; i < len; ++i) {
+		struct sway_layer_surface *lsurface;
+		wl_list_for_each(lsurface, &output->layers[i], link) {
+			json_object *layer = json_object_new_object();
+
+			json_object_object_add(layer, "namespace",
+				json_object_new_string(lsurface->layer_surface->namespace));
+
+			char *layer_name;
+			switch (lsurface->layer) {
+				case ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND:
+					layer_name = "background";
+					break;
+				case ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM:
+					layer_name = "bottom";
+					break;
+				case ZWLR_LAYER_SHELL_V1_LAYER_TOP:
+					layer_name = "top";
+					break;
+				case ZWLR_LAYER_SHELL_V1_LAYER_OVERLAY:
+					layer_name = "overlay";
+					break;
+			}
```

Comment:
Need to fix a maybe uninitializaed warning here (I can do this, just commenting for my own reminder)

Reply:
Forgot about this, thanks @RicArch97 for the reminder 😅